implementations/prometheus_benchmark/prometheus_benchmark_test.go

```diff
@@ -302,7 +302,7 @@ func BenchmarkPrometheusQueueWithJitter(b *testing.B) {
 							for k := 0; k < batchSize; k++ {
 								// Create labels for the metric
 								lbls := labels.Labels{
-									{Name: "__name__", Value: fmt.Sprintf("test_metric_%d", (writerID*1000+j+k)%10)},
+									{Name: "__name__", Value: fmt.Sprintf("test_metric_%d", rand.Uint32())},
```

Comment:
This is what bumped up the numbers on the benchmark, that modulus 10 meant I wasnt distributing the load properly.